src/ui/pages/ConnectedSites/ConnectedSites.tsx

```diff
@@ -143,38 +148,120 @@ function ConnectedSitesList({
   );
 }
 
+export function ConnectedSitesSearch({
```

Comment:
Why are we exporting?

Reply:
Looks like I wanted to keep it in a separate file initially but then changed my mind, removed `export` ✅